_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,986 @@
+to the [payjoin][topic payjoin] protocol, a privacy enhancing technique
+that makes it much more difficult for a third party to reliably
+associate inputs and outputs in a transaction with either the spender or
+receiver.  In February, he [proposed][feb payjoin] a serverless payjoin
+protocol than can be used even if the receiver doesn't operate an
+always-on HTTPS server on a public network interface.  In May, he
+[discussed][may payjoin] several advanced applications using payjoin,
+including variations of payment cut-through; for example, rather than
+Alice paying Bob, Alice instead pays Bob’s vendor (Carol), reducing a
+debt he owes her (or pre-paying for an expected future bill)---this
+saves block space and further improves privacy over standard payjoin.
+In August, he posted a [draft BIP][aug payjoin] for serverless payjoin
+which doesn't require both the spender and receiver to both be online at
+the same time (although each of them will need to come online at least
+once after the transaction is initiated before it can be broadcast).
+Throughout the year, he was a top contributor to the [payjoin
+development kit][jul pdk] (PDK) as well as the [payjoin-cli][dec
+payjoin] project that provides an add-on for creating payjoins with
+Bitcoin Core.
+
+{:#ark}
+Burak Keceli [proposed][may ark] a new joinpool-style protocol called
+Ark, where Bitcoin owners can opt-in to use a counterparty as a
+co-signer on all transactions within a certain time period. Owners can
+either withdraw their bitcoins onchain after the expiry of the timelock
+or instantly and trustlessly transfer them offchain to the counterparty
+before the timelock expires. The protocol provides a trustless
+single-hop, single-direction atomic transfer protocol from the owner to
+the counterparty for various uses such as mixing coins, making internal
+transfers, and paying LN invoices.  Concerns were raised about the high
+onchain footprint, large hot wallet requirements, and capital costs
+associated with this protocol compared to LN.  However, several
+developers remained enthusiastic about the proposed protocol and its
+potential to provide a simple and trustless experience for its users.
+
+## June
+
+{:#silentpayments}
+Josie Baker and Ruben Somsen [posted][jun sp] a draft BIP for [silent
+payments][topic silent payments], a type of reusable payment code that
+will produce a unique onchain address each time it is used, preventing
+[output linking][topic output linking].  Output linking can
+significantly reduce the privacy of users (including users not directly
+involved in a transaction). The draft goes into detail about the
+benefits of the proposal, its tradeoffs, and how software can
+effectively use it.
+
+<div markdown="1" class="callout" id="security">
+### 2023 summary<br>Security disclosures
+
+Optech reported on three significant security vulnerabilities this year:
+
+- [Milksad vulnerability in Libbitcoin `mx`][aug milksad]: a widely
```

Comment:
Wasn't it bx?

```suggestion
- [Milksad vulnerability in Libbitcoin `bx`][aug milksad]: a widely
```

Reply:
It was indeed: https://github.com/bitcoinbook/bitcoinbook/blob/develop/appdx-bx.asciidoc